core/ordering/cosipbft/controller/mod.go

```diff
@@ -162,6 +165,59 @@ func (minimal) Inject(flags cli.Flags, inj node.Injector) error {
 	return nil
 }
 
+// ClosableService is an extension of the ordering service that allows one to
+// close the service.
+type ClosableService interface {
```

Comment:
Being able to close a service seems a pretty common operation. Why not updating ordering.Service ?